api/events_test.go

```diff
@@ -23,7 +23,7 @@ func (fn *FakeNode) ID() string                            { return "node_id" }
 func (fn *FakeNode) Name() string                          { return "node_name" }
 func (fn *FakeNode) IP() string                            { return "node_ip" }
 func (fn *FakeNode) Addr() string                          { return "node_addr" }
-func (fn *FakeNode) Images() []*cluster.Image              { return nil }
+func (fn *FakeNode) Images(_ string) []*cluster.Image      { return nil }
```

Comment:
Sometimes you need only one images (for example for `docker history`)
`cluster.Images(name)[0]` can take much more time than `cluster.Image(name)`

That's why I kept the single image version.
Do you think it's not worth it ?